source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Deploying oVirt and Gluster Single node Hyperconverged
+---
+
+# Chapter 6: Deploying a Single Node oVirt and Gluster Hyperconverged
+
+## Pre-requisites
+
+* A single host with  Enterprise Linux 7 or oVirt Node. Refer [Enterprise Linux Hosts](../install-guide/chap-Enterprise_Linux_Hosts) or [oVirt Nodes](../install-guide/chap-oVirt_Nodes)
+
+* You must have at least 2 interfaces on the host, so that the frontend and backend traffic can be separated out. Having only one network will cause the engine monitoring, client traffic, gluster I/O traffic to all run together and interfere each other. To segregate the backend network, the gluster cluster is formed using the backend network addresses, and the nodes are added to the engine using the frontend network address.
+
+* You must have a fully qualified domain name prepared for your Engine and the host. Forward and reverse lookup records must both be set in the DNS. **The Engine should use the same subnet as the management network.**
+
+* You must have configured passwordless ssh between the host to itself. gdeploy uses Ansible playbooks and the ability to remotely execute commands is a pre-requisite.
+
+## Deploying on Enterprise Linux Hosts
+
+### Installing the Required Packages
+
+**Installing the packages on the first host**
+
+1. Subscribe to ovirt repos from http://resources.ovirt.org/pub/yum-repo/
+   For instance, to subscribe to oVirt 4.2 repo,
+
+        # yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release42.rpm
+
+2. Install gdeploy and cockpit-ovirt that will provide a UI for the installation of Hosted Engine. gdeploy is a wrapper tool around Ansible that helps to setup gluster volumes. vdsm-gluster is used to manage gluster from oVirt, and pulls in all the required gluster dependencies.
+     
+        # yum install gdeploy cockpit-ovirt-dashboard vdsm-gluster
+
+3. Install the oVirt Engine Virtual Appliance package for the Engine virtual machine installation:
+
+        # yum install ovirt-engine-appliance
+
+## Deploying on oVirt Node based Hosts
+
+**oVirt Node contains all the required packages to set up the hyperconverged environment.**
+Refer [oVirt Nodes](../install-guide/chap-oVirt_Nodes) for instructions on installing oVirt Node on your hosts. You can proceed to setting up the hyperconverged environment if you have 3 oVirt Node based hosts.
```

Comment:
1. Refer -> Refer to
2. Why do we need 3 nodes here?

Reply:
Thanks - copy & paste error.